### ISSUE TITLE
Make aria-label of Test timer button more accessible

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#8551]
## Area or Component

Conduct tests page - timer button

## Quick Summary

The timer button name is the same for every test card making it unclear which test card the timer is for.

## Accessibility Violation

[aria-name-missing-incorrect](https://docs.deque.com/issue-help/1.0.0/en/aria-name-missing-incorrect)

## Action Requested
Update `aria-label` like [we do here](https://github.com/CDCgov/prime-simplereport/blob/f443baacffeea64d435fdba29fd545dacc89de2e/frontend/src/app/testResults/viewResults/ResultsTable.tsx#L157) for the ["Start timer"](https://github.com/CDCgov/prime-simplereport/blob/c862a94015ce46f67918b528ad3c89e7f7b4e75b/frontend/src/app/testQueue/TestTimer.tsx#L273)  and "Reset timer" ([here](https://github.com/CDCgov/prime-simplereport/blob/c862a94015ce46f67918b528ad3c89e7f7b4e75b/frontend/src/app/testQueue/TestTimer.tsx#L291), [here](https://github.com/CDCgov/prime-simplereport/blob/c862a94015ce46f67918b528ad3c89e7f7b4e75b/frontend/src/app/testQueue/TestTimer.tsx#L314))

## Screenshots (optional)
<img width="1567" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd3dUQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--eae2c6301d9f40dd267461e5ab5289f6a7282715/Screenshot%202025-02-28%20at%2014.41.27.png" alt="Screenshot 2025-02-28 at 14.41.27.png" />

<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdzBUQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f7fc2aafa5acf44483098813447eb970cf638106/Screenshot%202025-02-28%20at%2014.41.52.png" alt="Screenshot 2025-02-28 at 14.41.52.png" />

## Source Code (optional)
See above
